.github/pull_request_template.md

```diff
@@ -28,6 +28,11 @@ Rule specific checks:
 - [ ] You have added a test case in `wdl-lint/tests/lints` that covers every
       possible diagnostic emitted for the rule within the file where the rule
       is implemented.
+- [ ] If you have implemented a new `Visitor` callback, you have also
```

Comment:
This was added after discussion with @peterhuene as its a common misstep when introducing a new visitor method.